x/feedist/msg_test.go

```diff
@@ -0,0 +1 @@
+package feedist
```

Comment:
pp: i think the concept is more general than just fees. How about `coindist`,`distgroup`?

Reply:
Everything is better than `feedist` :joy: 

I like both. Any preferences?

Reply:
no

Reply:
I will change to `distgroup`. There is [another pull request](https://github.com/iov-one/weave/pull/322) made to this one. I will wait for it to be merged and then rename the package.